s3-source-connector/src/integration-test/java/io/aiven/kafka/connect/s3/source/IntegrationBase.java

```diff
@@ -159,7 +159,7 @@ static <K, V> List<V> consumeMessages(final String topic, final int expectedMess
             consumer.subscribe(Collections.singletonList(topic));
 
             final List<V> recordValues = new ArrayList<>();
-            await().atMost(Duration.ofMinutes(2)).pollInterval(Duration.ofSeconds(1)).untilAsserted(() -> {
+            await().atMost(Duration.ofMinutes(10)).pollInterval(Duration.ofSeconds(5)).untilAsserted(() -> {
```

Comment:
10 minutes seems way too long to wait for an integration test

Reply:
Was testing with 10k recs and forgot to update this back. Changed it to 5, still it's big, but with awaitility, we don't really wait for 5 mins. Just in case, didn't want this test to become flaky on slow machines.